mkdocs_puml/plugin.py

```diff
@@ -40,7 +43,8 @@ class PlantUMLPlugin(BasePlugin):
         ('puml_url', Type(str, required=True)),
         ('num_workers', Type(int, default=8)),
         ('puml_keyword', Type(str, default='puml')),
-        ('verify_ssl', Type(bool, default=True))
+        ('verify_ssl', Type(bool, default=True)),
+        ('auto_dark', Type(bool, default=True))
```

Comment:
README.md says default is true, here, the default is `True`

Reply:
I assume your comment meant to point out that the README the default is `False` but the plugin is actually defaulting to `True`.

I updated it the plugin to default to `False` per the documentation, I assume this would be the desired behavior to avoid surprising existing users with double the # of images and conversion. What do you guys think?